library/src/scripts/components/PageHeading.tsx

```diff
@@ -12,26 +12,24 @@ import Heading from "@library/components/Heading";
 interface IPageHeading {
     title: string;
     children?: React.ReactNode;
-    backUrl?: string;
     className?: string;
     actions?: React.ReactNode;
+    includeBackLink?: boolean;
 }
 
 /**
  * A component representing a top level page heading.
  * Can be configured with an options menu and a backlink.
  */
 export default class PageHeading extends React.Component<IPageHeading> {
+    public defaultProps = {
```

Comment:
This needs to be `public static defaultProps: Partial<IPageHeading>`

The type is __necessary__ or it will break higher order components types.